### PR TITLE
v2.9.9

### PR DESCRIPTION
**Pull request by idastambuk**
## 2.9.9

- Chore: Update plugin.json keywords in [#310](https://github.com/grafana/timestream-datasource/pull/310)
- Update grafana-plugin-sdk-go and grafana-aws-sdk in [#309](https://github.com/grafana/timestream-datasource/pull/309)
- fix: linter complaints in [#308](https://github.com/grafana/timestream-datasource/pull/308)
- Bump path-to-regexp from 1.8.0 to 1.9.0 in [#303](https://github.com/grafana/timestream-datasource/pull/303)
- Docs: Updates and improvements in [#302](https://github.com/grafana/timestream-datasource/pull/302)
- Add dependabot for grafana/plugin-sdk-go in [#307](https://github.com/grafana/timestream-datasource/pull/301)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,15 @@
 
 All notable changes to this project will be documented in this file.
 
+## 2.9.9
```
Comment (iwysiu) [inline CHANGELOG.md:5]: shouldn't this include https://github.com/grafana/timestream-datasource/pull/311 ?

Comment (idastambuk) [reply CHANGELOG.md:5]: Oops, done!

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,16 @@
...
+
+- Fix errors in LongToWide transformation in [#311](https://github.com/grafana/timestream-datasource/pull/311)
+- Chore: Update plugin.json keywords in [#310](https://github.com/grafana/timestream-datasource/pull/310)
+- Update grafana-plugin-sdk-go and grafana-aws-sdk in [#309](https://github.com/grafana/timestream-datasource/pull/309)
+- fix: linter complaints in [#308](https://github.com/grafana/timestream-datasource/pull/308)
+- Bump path-to-regexp from 1.8.0 to 1.9.0 in [#303](https://github.com/grafana/timestream-datasource/pull/303)
+- Docs: Updates and improvements in [#302](https://github.com/grafana/timestream-datasource/pull/302)
+- Add dependabot for grafana/plugin-sdk-go in [#307](https://github.com/grafana/timestream-datasource/pull/301)
```
Comment (iwysiu) [inline CHANGELOG.md:13]:
```suggestion
- Add dependabot for grafana/plugin-sdk-go in [#307](https://github.com/grafana/timestream-datasource/pull/307)
```

**Review by iwysiu — APPROVED** [review]
You can merge https://github.com/grafana/timestream-datasource/pull/313 before this one and squish it in! (remember to add it to the changelog)

**Pull request merged**